app/routes/wow.ilvl-export-search.tsx

```
@@ -292,15 +292,25 @@ const IlvlExportSearchComponent = () => {
           type="number"
           value={formValues.ilvl.toString()}
           min={0}
-          onChange={(e) => setFormValues(prev => ({ ...prev, ilvl: parseInt(e.target.value) }))}
+          onChange={(e) =>
+            setFormValues((prev) => ({
+              ...prev,
+              ilvl: parseInt(e.target.value)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)